### ISSUE TITLE
"nearby" option fails to work again?

### ISSUE THREAD

**Issue by govvin** [osmlab/maproulette3#1524]
After completing a task, when a contributor selects the "nearby" tasks (without selecting any) in the vicinity, they are assigned other tasks that are much farther away.

This used to work as expected (getting nearby task, even without manually selecting one), and in the challenge I'm working on , there are no other higher priority tasks anyway. 

So the behavior appears to have changed recently.

**Comment by nrotstan** [comment]
Hi @govvin , I haven't been able to reproduce this so far. As you allude to, MR will give back the nearest high-priority task first, then the nearest medium-priority if there are no high-priority tasks remaining in the challenge, and finally the nearest low-priority task if there are no remaining medium-priority tasks. Just to clarify, those priorities are applied across the whole challenge, so it's not just higher priority tasks within a certain distance, but any higher priority task anywhere in the challenge (and then it takes you to whichever of those higher priority tasks is nearest to you). So if everything nearby is lower priority you could end up far away on a higher-priority task.

It's always possible to select a desired task on the Nearby map, of course, to be in control of which task you get next.

If you still think you're seeing incorrect behavior, can you let me know which challenge and an example task in the geographic area where you're seeing it? Thanks!

**Comment by govvin** [comment]
Was trying to replicate and document the same experience I had yesterday, but unable to do so. FYI, the [challenge](https://maproulette.org/challenge/9413) I tried this on doesn't have any set priority tasks.

Today, when I select the nearby option, I confirm that I now get one of the nearby tasks (but not necessarily the closest) without having to manually select any first.

Yesterday, when I filed the ticket, I am getting tasks from hundreds of kilometers away.  Perhaps yesterday was just a weird day.
